brompton/sass/style-child-theme.scss

```diff
@@ -15,7 +15,7 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 This theme, like WordPress, is licensed under the GPL.
 Use it to make something cool, have fun, and share what you've learned with others.
 
-Bromtpon is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc.
+Bromtpon is a child theme of Varia which is based on Underscores https://underscores.me/, (C) 2012-2019 Automattic, Inc.
```

Comment:
Looks like "Brompton" was already mispelled. Let's clean that up with this.

Reply:
I've fixed the misspelled theme name in https://github.com/Automattic/themes/pull/1666/commits/b4b9f8c3b39beb48b2c20e9167ea0629851a5e4b